tests/RestfulListTestCase.test

```diff
@@ -552,6 +552,20 @@ class RestfulListTestCase extends RestfulCurlBaseTestCase {
     $result = $handler->get('', $request);
     $this->assertEqual(count($result), 2, 'List filtered by the "author" entity metadata wrapper property, which maps to the "uid" DB column name, using the "NOT IN" operator.');
 
+    // Filter by an entity metadata wrapper property that is different from the
+    // DB column name using the 'NOT IN' operator.
+    $request = array(
+      'filter' => array(
+        'user' => array(
+          'value' => array(),
+          'operator' => 'NOT IN',
```

Comment:
Lets try this one with `IN` -- and I'd expect to get `this->assertEqual(count($result), 0,)`

Reply:
Since we skip the filter when the array is empty then we won't get this result.

Those queries seems problematic... I can add a special case for it though...